src/PAModel/Checksum/ChecksumMaker.cs

```diff
@@ -290,7 +291,7 @@ private static void ChecksumJson(Context ctx, IncrementalHash hash, JsonElement
                     else if (ctx.IsXmlDoubleEncoded && !string.IsNullOrWhiteSpace(str))
                     {
                         var parsedXML = XDocument.Parse(str);
-                        var xmlString = parsedXML.ToString(SaveOptions.None);
+                        var xmlString = parsedXML.ToString(SaveOptions.None).Replace("\r\n", "\n");
```

Comment:
this looks like the critical spot for checksum? catching the double escaped xml?

Reply:
yup